packages/contracts/contracts/plugin/examples/count/v1/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ui": "",
+  "change": "",
+  "pluginSetupABI": {
+    "prepareInstallation": "(address multiplyHelper, uint num)",
```

Comment:
I suggest you either add the full function signature `function prepareInstallation(address multiplyHelper, uint num)` or then as an array `["address multiplyHelper", "uint num"]`
The reason is that ethers and other libs either wanna have an array for ABI-De-/Encoding or then the full function signature. With this setup the developer either needs to parse it (which makes handling tuples hard)  or add all the time `function prepareInstallation`

Reply:
yes, thanks for the heads up,
the array ABI seems to be more useful for readability and to be used by consumers of the metadata
amended in `[0bfecf2](https://github.com/aragon/core/pull/208/commits/0bfecf27606444e4c03914913f0c90c19a2dcb69)`